README.md

```diff
@@ -1,4 +1,14 @@
-POC of new oVirt User Portal.
+# User Portal for oVirt
+Fancy looking UI for standard (non-admin) oVirt users.
+
+This project is meant to be full replacement for existing oVirt Basic User Portal.
```

Comment:
I would rephrase it to something like:
"This project aims to be a drop-in replacement of the existing Basic User Portal which includes selected functionality of the Power User Portal."